backup.txt

```diff
@@ -0,0 +1,68 @@
+<body>
+    <header class="headercontainer">
+      <div class="logo">Megha</div>
+          <div><i class="fa-solid fa-bars hambrgricon"></i></div>
+    </header>
+
+    <section class="main-section">
+      <div class="FirstShape">
+      </div>
+      <div class="first">
+        <img class="half-ellipse" src="/images/Ellipse11.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="Ellipse5" src="/images/Ellipse5.png" alt="shapes"/>
+      </div>
+      <div class="tiltsmallrect">
+        <img src="/images/Rectangle7.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="rectangle4" src="/images/Rectangle4.png" alt="shapes"/>
+      </div>
+      <div>
+        <img class="ellipse3" src="/images/Ellipse3.png" alt="shapes"/>
+      </div>
+    </section>
+
+    <div class="content">
+      <h1 class="boldtext">
+        I’m Megha<br />
+        Glad to see <br />
+        you!
+      </h1>
+      <p class="intro">
+        I’m a software developer! I can help you<br />
+        build a product , feature or website Look <br />
+        through some of my work <br />
+        and experience! If you like what you see and<br />
+        have a project you need coded, don’t <br />
+        hestiate to contact me.
+      </p>
+      <img class="Ellipse12" src="/images/Ellipse12.png" alt="shapes"/>
+      <img class="Ellipse13" src="/images/Ellipse13.png" alt="shapes"/>
+      <div>
+        <p class="socialmediatag">LET'S CONNECT</p>
+      </div>
+      <div>
+        <ul class="socialmediaicons">
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter fa-xl iconspace"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-linkedin-in fa-xl iconspace"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-medium fa-xl iconspace iconspace"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-github fa-xl iconspace "></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fab fa-angellist fa-xl iconspace"></i></a>
+          </li>
+        </ul>
+        </div>
+      <img class="Ellipse15" src="/images/Ellipse15.png" alt="shapes"/>
+      <img class="Ellipse4" src="/images/Ellipse4.png" alt="shapes"/>
+      <img class="rectangle5" src="/images/Rectangle5.png" alt="shapes"/>
+    </div>
```

Comment:
- [ ] Kindly, remove the unused code or add this file to `.gitignore`  file to make your project clean. 👌

Reply:
- [ ] [REQUIRED] I noticed you did not make changes as requested by the previous reviewer please make the change requested in order to make your project clean.

> * [ ]  Kindly, remove the unused code or add this file to `.gitignore`  file to make your project clean. ok_hand